### ISSUE TITLE
Cannot change the aria-label on startGrip and endGrip

### ISSUE THREAD

**Issue by Zerhar** [amcharts/amcharts5#112]
Cannot change the aria-label on startGrip and endGrip of the scrollbar. The following code did not work.

```
			scrollbar.startGrip.setAll({
				focusable: true,
				arialabel: "test",
			});

			scrollbar.endGrip.setAll({
				focusable: true,
				arialabel: "test",
			});
```

**Comment by martynasma** [comment]
There's currently no way to directly override `ariaLabel` (the setting uses camelCase btw) directly, as Scrollbar is designed to dynamically populate those values based on current selection range.

You can, however override it by modifying locale prompts:

```JavaScript
root.locale["From %1"] = "Foo %1";
root.locale["To %1"] = "Bar %1";
root.locale["From %1 to %2 %1"] = "Foo %1 bar %2";
```

The `%1` and `%2` are getting replaced by actual range values (e.g. start and end dates). If you do not need those in a prompt, you don't need to include them.

**Comment by Zerhar** [comment]
Alright, that worked beautifully. Thank you very much!.
